client.go

```diff
@@ -0,0 +1,32 @@
+package dp_redis
+
+import (
+	"github.com/go-redis/redis"
+	"time"
+)
+
+// Redis - structure for the redis client
+type Redis struct {
+	client *redis.Client
+	ttl    time.Duration
+}
+
+// Options - config options for the redis client
+type Options struct {
```

Comment:
Would `Config` be a more appropriate name? It's more descriptive about what purpose it is serving.

Reply:
I originally had this as `Config` only reason I changed it was to match the redis package wording as they call them `Options`.

Reply:
agreed - `Config` is more DP-style, too